torchci/lib/bot/retryBot.ts

```diff
@@ -120,6 +127,13 @@ async function retryCurrentWorkflow(
       return false;
     }
 
+    for (const flakyJobName of KNOWN_FLAKY_JOBS) {
+      // if the job is a known flaky one, we want to retry it
+      if (job.name.toLocaleLowerCase().includes(flakyJobName)) {
+        return true;
+      }
+    }
```

Comment:
Yeah, I'm trying to make this job eligible for retry even if its test step fails.  We don't retry that atm.

Reply:
I see.  In that case maybe a list name like "ALWAYS_RETRY" would make more sense then, since KNOWN_FLAKY_JOBS makes me think that you know it's probably flaky so it's ok to never retry it.  Doesn't really matter tho, since the comment explains